classes/Form/ConfigurationForm.php

```diff
@@ -97,7 +97,23 @@ public function generate()
                 ],
                 [
                     'type' => 'switch',
-                    'label' => $this->module->trans('Enable User ID tracking', [], 'Modules.GAnalytics.Admin'),
+                    'label' => $this->module->l('Enable Google Analytics 4'),
```

Comment:
`'label' => $this->module->trans('Yes', [], 'Modules.GAnalytics.Admin'),`
to maintain [PR 124](https://github.com/PrestaShop/ps_googleanalytics/pull/124/files#diff-5f981589f808927ab76592e5f9220ee730f4f05ecea0f9a10e0eef676546d5f2R100) and [PR 127](https://github.com/PrestaShop/ps_googleanalytics/pull/127/files#diff-5f981589f808927ab76592e5f9220ee730f4f05ecea0f9a10e0eef676546d5f2R100) works.